org/duet3d/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: org
+title: Duet3d
+---
+The Duet 3dprinter controller is an advanced 32 bit controller for RepRap and other 3d printers,
```

Comment:
This description text should probably be associated with the device, rather than the org.